### PR TITLE
Feature - compose and json serdes

### PR DESCRIPTION
**Pull request by zhenik**
Add JSON support, request by @karimacherradi

### CODE REVIEW

File: docker-compose/hive.env [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# metastore.uris for hive-server will be overriden, see docker-compose.yml
+HIVE_SITE_CONF_hive_metastore_uris=thrift://127.0.0.1:9083
+HIVE_SITE_CONF_hive_metastore_schema_verification=true
+HIVE_SITE_CONF_hive_execution_engine=mr
+HIVE_SITE_CONF_hive_support_concurrency=false
+HIVE_SITE_CONF_hive_driver_parallel_compilation=true
+HIVE_SITE_CONF_hive_metastore_warehouse_dir=s3a://hive/warehouse
+HIVE_SITE_CONF_hive_metastore_event_db_notification_api_auth=false
+HIVE_SITE_CONF_hive_server2_authentication=NOSASL
+# csv support from presto todo https://github.com/prestosql/presto/issues/2678#issuecomment-580255387
```
Comment (m88614) [inline docker-compose/hive.env:16]: Should be it's own issue, no?

File: docker-compose/hive.env [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# ConnectionURL for hive-server will be overriden, see docker-compose.yml
+HIVE_SITE_CONF_javax_jdo_option_ConnectionURL=jdbc:postgresql://database:5432/metastore
+HIVE_SITE_CONF_javax_jdo_option_ConnectionDriverName=org.postgresql.Driver
+HIVE_SITE_CONF_javax_jdo_option_ConnectionUserName=hive
+HIVE_SITE_CONF_javax_jdo_option_ConnectionPassword=hive
+HIVE_SITE_CONF_datanucleus_autoCreateSchema=false
+# metastore.uris for hive-server will be overriden, see docker-compose.yml
```
Comment (m88614) [inline docker-compose/hive.env:7]: Might just be me not understanding it fully, but where are the uris overriden? Can't see anything in docker-compose that overrides it?

File: docker-compose/hive.env (hunk 1 above)
Comment (zhenik) [reply docker-compose/hive.env:16]: What do u mean here? It is presto related issue, and I see it was fixed so far. I would prefer to have ref, maybe remove todo (WDYT?)

File: docker-compose/hive.env (hunk 2 above)
Comment (zhenik) [reply docker-compose/hive.env:7]: sorry, it was copy-paste from [the previous configuration](https://github.com/zhenik-poc/big-data-stack-practice/blob/master/hive.env#L7). I will add hive in server mode to `docker-compose.yml` to have actually full picture how to run hive image in two different modes (metastore and hive-server)

File: docker-compose/hive.env (hunk 1 above)
Comment (m88614) [reply docker-compose/hive.env:16]: I was just talking about the fact that it says todo. If it is fixed we should just remove the 'todo' part of it.

File: docker-compose/hive.env (hunk 2 above)
Comment (m88614) [reply docker-compose/hive.env:7]: Aha got it!

File: Dockerfile [hunk 3]
```diff
@@ -27,6 +27,9 @@ RUN \
     && curl -s -L $HIVE_DOWNLOAD | tar -xz --transform s/apache-hive-$HIVE_VERSION-bin/hive/ -C /opt/ \
     #Download postgres jdbc driver
     && curl -s -L https://jdbc.postgresql.org/download/postgresql-$POSTGRES_JDBC_VERSION.jar -o $HIVE_HOME/lib/postgresql-jdbc.jar \
+    #Downliad json serializer/deserializer
```
Comment (m88614) [inline Dockerfile]:
```suggestion
    #Download json serializer/deserializer
```

**Comment by zhenik** [comment]
Require new IMAGE release @fredrikhgrelland , but first, check #30

**Pull request merged**